cmd/iam.go

```diff
@@ -1155,7 +1271,20 @@ func (sys *IAMSys) removeGroupFromMembershipsMap(group string) {
 
 // NewIAMSys - creates new config system object.
 func NewIAMSys() *IAMSys {
+	// Check global server configuration to determine the type of
+	// users system configured.
+
+	// The default users system
+	var utype UsersSysType
+	switch {
+	case globalServerConfig.LDAPServerConfig.ServerAddr != "":
+		utype = LDAPUsersSysType
+	default:
+		utype = MinIOUsersSysType
+	}
```

Comment:
was this decided that we will either support Multi-User or LDAP - since this contradicts with WebIdentity implementation i.e multi-user works even with WebIdentity enabled.

Reply:
Yes, it was decided for LDAP support. The code should be compatible with existing WebIdentity implementation - preserving the current behaviour of allowing minIO local users with webidentity users - but we need to discuss if this the behaviour going forward.